.github/component_owners.yml

```diff
@@ -4,6 +4,7 @@ components:
   src/OpenFeature.Contrib.Hooks.Otel:
     - bacherfl
     - toddbaert
+    - askpt
```

Comment:
@askpt I hope you don't mind I've added you here. This will add you to PRs for this component automatically, [once you're an org member](https://github.com/open-feature/community/pull/309).